tests/test_transform.py

```diff
@@ -197,9 +197,9 @@ def test_set_basis(self):
         """Tests the basis setter method."""
         grid = ConcentricGrid(L=4, M=2, N=1)
 
-        basis_20 = FourierZernikeBasis(L=-1, M=2, N=0)
```

Comment:
do you know why these were -1 before?

Reply:
oh was it to set it to the default based on the indexing?

Reply:
yeah `L=-1` meant `M if spectral_indexing=="ansi" else 2*M` but it wasn't really used anywhere and its better to be explicit.